scripts/test_metadata.sh

```diff
@@ -0,0 +1,48 @@
+export KEY="tf1" # wasm1hj5fveer5cjtn4wd6wstzugjfdxzl0xpvsr89g
```

Comment:
Where is this script called?

Reply:
This is just a testing script to ensure the chain starts up correctly. Purely for development, not CI